doc/userguide/output/eve/eve-json-output.rst

```diff
@@ -460,6 +460,9 @@ The ``community-id-seed`` option specifies a unsigned 16 bit value that
 is used a seed to the hash that is calculated for the ``community-id``
 output. This must be set to the same value on all tools that output this
 record.
+The ``encrypted-traffic-metadata`` option is boolean. If set to ``true`` it 
```

Comment:
Can you expand this to include what the feature actually is, e.g. with a link to the doc linked in the PR for example.

Reply:
Added some more explanation of the feature. I did not add the link because I was not sure if that link would be persistent (I do not work for Cisco). I can add it if you think it is helpful though.